modules/cnv-cdi-supported-operations-matrix.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-cdi-supported-operations.adoc
```

Comment:
This document is almost identical to this one:  modules/cnv-cdi-scratch-space-support-matrix.adoc
Are you sure we need both?

Reply:
You're right, they are clones with different names. I thought I had removed 'modules/cnv-cdi-scratch-space-support-matrix.adoc' from the repo (it isn't referenced in any assemblies) but I must have been mistaken. 
This one' modules/cnv-cdi-supported-operations-matrix.adoc' is the one to keep.